azure-mgmt-containerservice/src/main/java/com/azure/management/containerservice/CloudError.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.management.containerservice;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The CloudError model.
+ */
+@Fluent
+public final class CloudError {
```

Comment:
seems one change in azure core is to generalize exception, not using CloudError, is this fluent layer?

Reply:
This is a generated class, which is defined by [swagger](https://github.com/azure/azure-rest-api-specs/blob/1c3a60403422ca8c73df7eb7c65ae2a3c90bf152/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-08-01/managedClusters.json#L2269). They use spatial cloud error rather than common one.

Reply:
This could be a generator bug. CloudError and CloudException is defined in Core Management. Hence generator does not need to generate CloudError class.